main.go

```diff
@@ -12,9 +12,16 @@ import (
 func main() {
 	r := mux.NewRouter()
 
+	hub := newHub()
+	go hub.run()
+
 	r.HandleFunc("/problems", getProblems).Methods("GET")
 	r.HandleFunc("/problems/{id}", getProblem).Methods("GET")
 
+	r.HandleFunc("/ws", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
¿no tendremos un archivo solo de rutas?

Reply:
idk, I think we could manage to make `main.go` the config and routes file. Doesn't seem like a bad practice to me. But we can change it as it grows.